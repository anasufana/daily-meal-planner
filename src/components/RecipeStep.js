import React from 'react';
import '../css/components/MealCaloriesDonut.css';

const RecipeStep = props => (
  <div className="steps-container">
    <h2 className="step-title">Step <span className="step-number">{ props.details.number }</span>:</h2>
    <p className="step-description">{ props.details.step }</p>
  </div>
);

export default RecipeStep;