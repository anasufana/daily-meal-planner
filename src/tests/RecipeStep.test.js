/* global it, expect, jest */

import React from 'react';
import renderer from 'react-test-renderer';
import RecipeStep from '../components/RecipeStep';

const details = {
  details: {
    number: 1,
    step: 'Preheat the oven to 200 degrees F.'
  },
};

it('renders correctly', () => {
  const tree = renderer
    .create(<RecipeStep
      details={details.details}
    />).toJSON();
  expect(tree).toMatchSnapshot();
});
