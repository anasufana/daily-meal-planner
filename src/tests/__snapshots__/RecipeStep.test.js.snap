// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="steps-container"
>
  <h2
    className="step-title"
  >
    Step 
    <span
      className="step-number"
    >
      1
    </span>
    :
  </h2>
  <p
    className="step-description"
  >
    Preheat the oven to 200 degrees F.
  </p>
</div>
`;
